test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -339,8 +339,12 @@ function compute-etcd-events-params {
 function compute-kube-apiserver-params {
 	local params="${APISERVER_TEST_ARGS:-}"
 	params+=" --insecure-bind-address=0.0.0.0"
-	params+=" --etcd-servers=http://127.0.0.1:2379"
-	params+=" --etcd-servers-overrides=/events#${EVENT_STORE_URL}"
+	if [ -z "${ETCD_SERVERS}" ]; then
```

Comment:
We prefer using double brackets as it is recommended in our bash scripts (IIRC for better predictability).

Nit: Maybe use "${ETCD_SERVERS:-}", like the pattern in other places?